docs/conf.py

```diff
@@ -85,6 +56,7 @@ def _check_objects_are_documented():
     "examples_dirs": "../examples",
     "gallery_dirs": "_gallery",
     "backreferences_dir": "_gallery/backreferences",
+    "exclude_implicit_doc": {r"Resource"},
```

Comment:
Exclude some more, e.g. base classes.
```suggestion
    "exclude_implicit_doc": {"WgpuRenderer", "Resource", "WorldObject", "Geometry", "Material", "Controller", "Camera", "show", "Display", "Group", "Scene", "Light"},
```